allennlp/data/dataset_readers/semantic_role_labeling.py

```diff
@@ -14,6 +15,54 @@
 
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
+START_TOKEN = "[CLS]"
```

Comment:
since these are only used once, is there value in pulling them out into module-level variables?